sshuttle/server.py

```diff
@@ -192,6 +192,8 @@ def try_send(self):
             port = 53
             family, sockaddr = self._addrinfo(peer, port)
         else:
+            _, peer = self.to_nameserver
+            port = 53
```

Comment:
Unfortunately, we don't actually know that the port number is 53. Also peer isn't a simple text string, it is a list of tuples returned by `getaddrinfo`.

A better option, maybe we need to preserve the values of `peer` and `port` in `self` after line 174.

Does this make sense?